mergify_engine/engine/actions_runner.py

```diff
@@ -57,6 +57,44 @@
 
 """
 
+COMMIT_MESSAGE_MODE_MERGE_DEPRECATION_GHES = """
+:bangbang: **Action Required** :bangbang:
+
+> **The configuration uses the deprecated `commit_message` mode of the merge action.**
+> This option will be removed on a future version.
+> For more information: :ref: `doc.mergify.io/actions/merge/`
```

Comment:
This is markdown text posted in a PR, not RST documentation. You need a proper link formatted in Markdown.